sfcasts/factory-families.md

```diff
@@ -98,8 +98,7 @@ Let's try it out! Spin over to your terminal and run:
 php bin/console app:game:play -c up-up-down-down-left-right-left-right-b-a-start
 ```
 
-And don't forget to pass the cheat code! Hey! Look at that! There's our
-`Ultimate cheat code activated!` message. And if we battle... we *won* in just
-*two rounds*! Cheat codes for the win!
+Hey! Look at that! There's our `Ultimate cheat code activated!` message. And if we
```

Comment:
Felt weird to say the cheat code then, right after, say "And don't forget to pass the cheat code!" but what do you think @MolloKhan?